### ISSUE TITLE
Feature Request: mouse-2 password bar 

### ISSUE THREAD

**Issue by danrobi11** [status-im/status-desktop#2702]
Hello. Im a Emacs user. Emacs is my WM with EXWM. 

Copy | paste doesnt work as intended. Therefore, only mouse-2 (right-click) does work with external apps. 

I cant paste my password in the password bar at launch. 
I need the mouse-2 to work in that bar to be able to paste my password.

Thanks!

**Comment by michaelsbradleyjr** [comment]
It seems that, in general, in the Status Desktop's app's input areas, right-clicking (mouse-2) isn't handled, i.e. nothing happens when you right click the password input on the login screen, the message input bar in chat, the chat search bar, etc.

I've tried on macOS (with control+click) and you're on Linux (I'm assuming, based on your setup), so it's clearly something we need to handle in the app, i.e. it's not a limitation of a particular OS/environment.

Thanks for submitting this issue, it's something the desktop team can look into soon, though I can't make any guarantees re: timeframe.

**Comment by danrobi11** [comment]
@michaelsbradleyjr 

I found out that `mouse button(Keys.BTN_MIDDLE)` does paste clipboard.
That's a good workaround

**Comment by michaelsbradleyjr** [comment]
> I found out that `mouse button(Keys.BTN_MIDDLE)` does paste clipboard.
> That's a good workaround

Nice, thanks for sharing the workaround!

**Comment by Khushboo-dev-cpp** [comment]
@iurimatias do we want the use too see a modal with options such copy ![image.png](https://images.zenhubusercontent.com/610aa02b14ef6a62cbf14bf1/b8ee210b-0929-4a99-97e4-95829985e0b1)

@simonricoo ?

**Comment by Khushboo-dev-cpp** [comment]
I checked and ctrl+v and cmd+v does work on linux and macOS

**Comment by simonricoo** [comment]
Yes, we should offer a "past" option when you have something in you clipboard on the right click on all input fields. I would skip the other options discord offers. @Khushboo-dev-cpp
